WorldModel/WorldModel/Core/CoreCommands.aslx

```diff
@@ -913,35 +913,26 @@
   <command name="wait" pattern="[wait]">msg (Template("DefaultWait"))</command>
   <command name="xyzzy" pattern="[xyzzy]">msg (Template("DefaultXyzzy"))</command>
   
-  <!-- Modified to suppress turn scripts -->
   <command name="help" pattern="[help]">
     <script>
       msg (Template("DefaultHelp"))
       game.suppressturnscripts = true
     </script>
   </command>
 
-  <!-- Modified by KV to resolve issue with empty divOutput when saving online with this command. -->
   <command name="save">
     <pattern type="string">^save$</pattern>
     <script>
-      if (HasAttribute(game, "questplatform")) {
-        if (game.questplatform = "desktop") {
-           request(RequestSave, "")
-        }
-        else {
-          JS.saveGame ()
-        }
+      if (IsDesktop()){}

```

Comment:
The open and close braces here looks like an error?

Reply:
> The open and close braces here looks like an error?

This is proof that I should switch back to Notepad++. I like VS Code, but I apparently need to "practice" with it before using it for important things.

I just fixed this.

Reply:
> I'm not sure why we should have separate code for desktop and web within an ASLX file. Shouldn't `RequestSave` do the right thing instead?

I agree.

I looked at that code a few times before doing it this way, and I don't know what that C# code is doing. If you can make `RequestSave` handle it, that sounds much better than this.

Reply:
Is there an Issue for this one?

Reply:
10-4

https://github.com/textadventures/quest/issues/1200